src/main/java/org/mskcc/oncokb/curation/service/CancerTypeService.java

```diff
@@ -128,6 +142,202 @@ public Optional<CancerType> findByMainTypeAndSubtypeIsNull(String mainType) {
         return cancerTypeRepository.findByMainTypeAndSubtypeIsNull(mainType);
     }
 
+    @Transactional(readOnly = true)
+    public List<CancerType> getRelevantCancerTypes(String levelOfEvidence, List<RelevantCancerTypeQuery> cancerTypeQueries) {
+        boolean isLevelBased = levelOfEvidence != null;
+        RelevantCancerTypeDirection direction = levelOfEvidence != null && levelOfEvidence.equals(LevelOfEvidenceEnum.LEVEL_DX1.getValue())
```

Comment:
Really just nit picking, since you already defined `isLevelBased`, might as well just use here.